packages/react-test-renderer/src/__tests__/ReactShallowRenderer-test.js

```diff
@@ -890,8 +888,6 @@ describe('ReactShallowRenderer', () => {
       componentWillMount() {
         this.setState({foo: 'bar'}, () => mockFn());
         this.setState({foo: 'foobar'}, () => mockFn());
-
-        callbackQueueLength = shallowRenderer._updater._callbacks.length;
```

Comment:
I didn't find a way to replace this specific test (that the callback queue was actually growing). However, the test that ensures that the `mockFn` was called twice – do you think this is enough?

Reply:
Yes